[2022-08-03 16:40:04,494] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.say_Hi 2022-08-03T16:30:00+00:00 [queued]>
[2022-08-03 16:40:04,986] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.say_Hi 2022-08-03T16:30:00+00:00 [queued]>
[2022-08-03 16:40:04,998] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 16:40:04,999] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2022-08-03 16:40:05,021] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 16:40:05,096] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): say_Hi> on 2022-08-03T16:30:00+00:00
[2022-08-03 16:40:05,216] {standard_task_runner.py:51} INFO - Started process 21542 to run task
[2022-08-03 16:40:05,223] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'my_simple_dag', 'say_Hi', '2022-08-03T16:30:00+00:00', '--job-id', '1283', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_simple_dag.py', '--cfg-path', '/tmp/tmpebsd3mgh']
[2022-08-03 16:40:05,239] {standard_task_runner.py:76} INFO - Job 1283: Subtask say_Hi
[2022-08-03 16:40:05,500] {logging_mixin.py:103} INFO - Running <TaskInstance: my_simple_dag.say_Hi 2022-08-03T16:30:00+00:00 [running]> on host b4dcda255553
[2022-08-03 16:40:07,008] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_simple_dag
AIRFLOW_CTX_TASK_ID=say_Hi
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T16:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-03T16:30:00+00:00
[2022-08-03 16:40:07,059] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-08-03 16:40:07,072] {bash.py:158} INFO - Running command: echo "Hi!!"
[2022-08-03 16:40:07,100] {bash.py:169} INFO - Output:
[2022-08-03 16:40:07,154] {bash.py:173} INFO - Hi!!
[2022-08-03 16:40:07,179] {bash.py:177} INFO - Command exited with return code 0
[2022-08-03 16:40:08,243] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=my_simple_dag, task_id=say_Hi, execution_date=20220803T163000, start_date=20220803T164004, end_date=20220803T164008
[2022-08-03 16:40:14,581] {local_task_job.py:170} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-08-03 16:40:14,637] {process_utils.py:95} INFO - Sending Signals.SIGTERM to GPID 21542
[2022-08-03 16:40:23,320] {taskinstance.py:1214} ERROR - Received SIGTERM. Terminating subprocesses.
[2022-08-03 16:40:25,658] {bash.py:185} INFO - Sending SIGTERM signal to bash process group
[2022-08-03 16:40:27,670] {process_utils.py:61} INFO - Process psutil.Process(pid=21542, status='terminated', exitcode=1, started='16:40:04') (21542) terminated with exit code 1
[2022-08-03 16:40:27,672] {local_task_job.py:118} INFO - Task exited with return code 1
